react/features/video-quality/subscriber.js

```diff
@@ -235,7 +238,13 @@ function _updateReceiverVideoConstraints({ getState }) {
 
         if (visibleRemoteParticipants?.size) {
             visibleRemoteParticipants.forEach(participantId => {
-                const sourceName = getTrackSourceNameByMediaTypeAndParticipant(tracks, MEDIA_TYPE.VIDEO, participantId);
+                let sourceName;
+
+                if (remoteScreenShares.includes(participantId)) {
+                    sourceName = getTrackSourceNameByFakeScreenShareParticipant(tracks, participantId);
```

Comment:
I just realized we don't even need this if a fake SS participant's id is the sourceName already